skrf/network.py

```diff
@@ -539,6 +539,51 @@ def __pow__(self, other: 'Network') -> 'Network':
         else:
             return cascade(self, other)
 
+    def __rshift__(self, other: 'Network') -> 'Network':
+        """
+        Cascade two 4-port networks with "1=>2/3=>4" port numbering.
+    Note
+    ----
+    connection diagram::
+
+              A               B
+           +---------+   +---------+
+          -|0        1|---|0        1|-
+          -|2        3|---|1        3|-
+          ...       ... ...       ...
+          -|2N-4  2N-3|---|2N-4  2N-3|-
+          -|2N-2  2N-1|---|2N-2  2N-1|-
+           +---------+   +---------+
+
+        Returns
+        -------
+        ntw : :class:`Network`
+            Cascaded Network
+
+        See Also
+        --------
+        cascade
+
+        """
+        check_nports_equal(self, other)
+        check_frequency_exist(self)
+        (n,_) = shape(self.s[0])
+        if (n / 2) != (n // 2):
+            raise ValueError("Operator >> requires an even number of ports.")
+
+        ix_old = list(range(n))
+        n_2    = n//2
+        n_2_1  = list(range(n_2))
+        ix_new = list(sum(zip(n_2_1, list(map((lambda x: x + n_2), n_2_1))), ()))
```

Comment:
Crazy! Let's hope it works.

Reply:
Oh, yeah, I should've asked first whether you folks are okay w/ the functional style; sorry. :(
Happy to expand that out into a more imperative style if that's your preference.

Reply:
We do not enforce a specific style. Eventually, a comment explaining the thing?

Reply:
Great !
Is this similar to:
```Python
N = ntw.nports
old_order = list(range(0, N, 2)) + list(range(1, N, 2))
new_order = list(range(N))
ntw.renumber(old_order, new_order)
```

Reply:
No, it's fine. I did not mean the "crazy" in a negative sense. Still, a comment would not hurt here ;)